.github/workflows/versionincrement.yaml

```diff
@@ -0,0 +1,78 @@
+        default: warning
+        type: choice
+        options:
+          - info
+          - warning
+          - debug
+
+jobs:
+  plugin-version-increment-sync:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        repo: [common-utils, job-scheduler, ml-commons, performance-analyzer, performance-analyzer-rca, security, alerting, index-management, sql, observability, asynchronous-search, k-NN, anomaly-detection, cross-cluster-replication, dashboards-reports, geospatial, notifications]
+        branch: [main, 2.x, '2.1', '2.2']
+    env:
+      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+    steps:
+      - name: Check out OpenSearch repo
+        uses: actions/checkout@v3
+        with:
+          repository: opensearch-project/OpenSearch
+          ref: ${{ matrix.branch }}
+      - name: Fetch OpenSearch version
+        run: |
+          OPENSEARCH_VERSION=$(cat buildSrc/version.properties | grep opensearch | cut -d= -f2 | grep -oE '[0-9.]+')
+          OPENSEARCH_VERSION=$OPENSEARCH_VERSION-SNAPSHOT
+          echo "OPENSEARCH_VERSION=$OPENSEARCH_VERSION" >> $GITHUB_ENV
+          echo "OPENSEARCH_VERSION_NUMBER=$OPENSEARCH_VERSION" | cut -f1 -d"-" >> $GITHUB_ENV
+      - name: Check out plugin repo
+        uses: actions/checkout@v3
+        with:
+          repository: opensearch-project/${{ matrix.repo }}
+          ref: ${{ matrix.branch }}
+      - name: Version Increment plugin repo
+      # If condition for plugins with Gradle Project in subfolders.
+        run: |
+          echo "OpenSearch Core repo version on branch ${{ matrix.branch }} is ${{ env.OPENSEARCH_VERSION_NUMBER }}"
+          if [ ${{ matrix.repo }} == "dashboards-reports" ]; then
+            cd reports-scheduler
+            ./gradlew versionIncrement -DnewVersion=${{ env.OPENSEARCH_VERSION }}
+          elif [ ${{ matrix.repo }} == "observability" ]; then
+            cd opensearch-observability
+            ./gradlew versionIncrement -DnewVersion=${{ env.OPENSEARCH_VERSION }}
+          elif [ ${{ matrix.repo }} == "notifications" ]; then
+            cd notifications
+            ./gradlew versionIncrement -DnewVersion=${{ env.OPENSEARCH_VERSION }}
+          else
+            ./gradlew versionIncrement -DnewVersion=${{ env.OPENSEARCH_VERSION }}
+          fi
+      - name: Create Pull Request
+        id: cpr
+        uses: peter-evans/create-pull-request@v4
+        with:
+          token: ${{ secrets.CIBOTAUTOPR }}
+          author: opensearch-ci-bot <opensearch-infra@amazon.com>
+          commit-message: |
+            Version Increment.
+
+            Signed-off-by: opensearch-ci-bot <opensearch-infra@amazon.com>
+          delete-branch: true
+          branch: create-pull-request/${{ env.OPENSEARCH_VERSION }}
+          title: '[AUTO] Version Increment to ${{ env.OPENSEARCH_VERSION }}'
+          body: |
+            - Version Increment for **${{ matrix.repo }}** plugin to **${{ env.OPENSEARCH_VERSION }}**.
```

Comment:
Incremented version to ...

You don't need the plugin name because it's in the plugin repo.